enterprise/server/gcplink/gcplink.go

```diff
@@ -0,0 +1,83 @@
+package gcplink
+
+import (
+	"fmt"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/buildbuddy-io/buildbuddy/enterprise/server/util/keystore"
+	"github.com/buildbuddy-io/buildbuddy/server/environment"
+	"github.com/buildbuddy-io/buildbuddy/server/util/cookie"
+	"github.com/buildbuddy-io/buildbuddy/server/util/status"
+
+	ctxpb "github.com/buildbuddy-io/buildbuddy/proto/context"
+	skpb "github.com/buildbuddy-io/buildbuddy/proto/secrets"
+	requestcontext "github.com/buildbuddy-io/buildbuddy/server/util/request_context"
+)
+
+const (
+	Issuer                      = "https://accounts.google.com"
+	scope                       = "https://www.googleapis.com/auth/cloud-platform"
+	paramName                   = "link_gcp_for_group"
+	cookieName                  = "link-gcp-for-group"
+	cookieDuration              = 1 * time.Hour
+	refreshTokenEnvVariableName = "CLOUDSDK_AUTH_REFRESH_TOKEN"
+	authAccountEnvVariableName  = "CLOUDSDK_AUTH_ACCOUNT"
```

Comment:
would it make sense to prefix these with `GOOGLE`? CLOUDSDK might be ambiguous if we support other cloud providers

Reply:
wanna add that link here as a comment explaining that these must match so that gcloud will pick them up?

Reply:
Added a comment explaining what to do with it, but it's actually not the variable from the link (the naming is just derived from it).